shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -538,7 +537,7 @@ void* TizenRendererEvasGL::OnProcResolver(const char* name) {
   GL_FUNC(glTexStorage3DMultisample)
 #undef GL_FUNC
 
-  FT_LOGD("Could not resolve: %s", name);
+  FT_LOGE("Could not resolve: %s", name);
```

Comment:
Shouldn't this be `FT_LOGI` or `FT_LOGW`?

@bbrto21 @bwikbs How do you think of this change?

Reply:
I've changed this to LOGW as you suggested.

Reply:
That seems right. Even if there is no procedure, it is not a direct error situation because ``skia`` handles fallback logic.